src/Directory.Packages.props

```diff
@@ -4,11 +4,12 @@
 		<PackageVersion Include="DotNet.ReproducibleBuilds" Version="1.1.1" />
 		<PackageVersion Include="FluentAssertions" Version="6.7.0" />
 		<PackageVersion Include="IdentityModel.OidcClient" Version="5.0.0" />
+		<PackageVersion Include="FluentValidation " Version="11.9.0" />
 		<PackageVersion Include="MSTest.TestAdapter" Version="2.2.9" />
 		<PackageVersion Include="MSTest.TestFramework" Version="2.2.9" />
 		<PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
-		<PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.1.0" />
-		<PackageVersion Include="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0"  />
+		<PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.9.2" />
```

Comment:
Why is this update needed?
It will require Visual Studio 17.9, so 17.8 and earlier will no longer be supported.

Reply:
I thought it just brings new code analysis features

Reply:
@xperiandri Nope, this has an impact on the minimum supported VS version